ldi-core/request-executor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/requestexecutor/executor/edc/services/MemoryTokenService.java

```diff
@@ -16,11 +16,15 @@ public class MemoryTokenService implements TokenService {
 
 	private EdcToken token;
 
+	private final MemoryTokenServiceLifecycle lifecycle = new MemoryTokenServiceLifecycle();
```

Comment:
I am hesitating if this lifecycle should not be injected into the class instead of creating it here?

Reply:
fixed